### PR TITLE
Add Delta Dental

### PR DESCRIPTION
**Pull request by Rabadash8820**
Delta Dental is a US dental insurance provider.

I was a little unsure how to handle this one. It appears that Delta Dental operates in [most US states/territories](https://www.deltadental.com/us/en/about-us/delta-dental-member-companies.html), but each one has its own domain. Credentials do not seem to be shared between domains (e.g., my `deltadentalwa.com` creds don't work on `deltadentalma.com`); however, all sites have very similar login forms with almost identical branding and messaging. The `modahealth.com` domain (for Oregon) is the most different, and is apparently from an insurance company that was [acquired by Delta Dental](https://dfr.oregon.gov/business/reg/insurer/mergers/pages/public-hearing-modahealth-deltadental.aspx) in 2018. If you'd prefer, I could break up this single JSON entry into separate ones for each domain.

Presumably (I didn't test this with all domains), the experience for enabling 2FA is the same for all of these sites since they're all managed by Delta. Here is the experience for `deltadentalwa.com`:

image

### CODE REVIEW

File: entries/d/deltadental.com.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "deltadentalks.com",
+      "deltadentalok.org",
+      "deltadentalpr.com",
+      "kentucky.deltadental.com",
+      "northcarolina.deltadental.com",
+      "tn.deltadental.com",
+      "www.deltadentalaz.com",
+      "www.deltadentalco.com",
+      "www.deltadentalct.com",
+      "www.deltadentalia.com",
```
Comment (Carlgo11) [inline entries/d/deltadental.com.json]: Can you confirm that these work?
I'm apparently blocked from accessing them.

File: entries/d/deltadental.com.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "www.deltadentalid.com",
+      "www.deltadentalil.com",
+      "www.deltadentalin.com",
```
Comment (Carlgo11) [inline entries/d/deltadental.com.json]: Please confirm that this domain works.

File: entries/d/deltadental.com.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "www.deltadentalins.com",
+      "www.deltadentalma.com",
+      "www.deltadentalmi.com",
+      "www.deltadentalmn.org",
+      "www.deltadentalmo.com",
+      "www.deltadentalne.org",
+      "www.deltadentalnj.com",
```
Comment (Carlgo11) [inline entries/d/deltadental.com.json]: This website times out for me

File: entries/d/deltadental.com.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      "www.deltadentalri.com",
+      "www.deltadentalsc.com",
+      "www.deltadentalsd.com",
+      "www.deltadentalva.com",
+      "www.deltadentalwa.com",
+      "www.deltadentalwi.com",
+      "www.deltadentalwy.org",
+      "www.hawaiidentalservice.com",
+      "www.modahealth.com",
+      "www.nedelta.com"
```
Comment (Carlgo11) [inline entries/d/deltadental.com.json]: We prefer if you don't list the subdomains unless it is needed to segregate services.
```suggestion
      "deltadentalaz.com",
      "deltadentalco.com",
      "deltadentalct.com",
      "deltadentalia.com",
      "deltadentalid.com",
      "deltadentalil.com",
      "deltadentalin.com",
      "deltadentalins.com",
      "deltadentalma.com",
      "deltadentalmi.com",
      "deltadentalmn.org",
      "deltadentalmo.com",
      "deltadentalne.org",
      "deltadentalnj.com",
      "deltadentalnm.com",
      "deltadentaloh.com",
      "deltadentalri.com",
      "deltadentalsc.com",
      "deltadentalsd.com",
      "deltadentalva.com",
      "deltadentalwa.com",
      "deltadentalwi.com",
      "deltadentalwy.org",
      "hawaiidentalservice.com",
      "modahealth.com",
      "nedelta.com"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @Rabadash8820!
Some of the domains you listed don't resolve for me, could you please verify that they work?

Also, SVG icons are preferred to PNGs. Here's one you can use instead:

File: entries/d/deltadental.com.json (hunk 1 above)
Comment (Rabadash8820) [reply entries/d/deltadental.com.json]: Yeah those two resolve for me. Maybe you can use a US-based VPN or something to see them?

File: entries/d/deltadental.com.json (hunk 2 above)
Comment (Rabadash8820) [reply entries/d/deltadental.com.json]: Yep these all resolve for me

File: entries/d/deltadental.com.json (hunk 1 above)
Comment (Carlgo11) [reply entries/d/deltadental.com.json]: I tried but they seem to block VPNs. If they work for you then it's fine.

File: entries/d/deltadental.com.json (hunk 3 above)
Comment (Rabadash8820) [reply entries/d/deltadental.com.json]: Hm, the New Jersey (`*nj`) one? These all successfully load for me

**Comment by Rabadash8820** [comment]
> Also, SVG icons are preferred to PNGs. Here's one you can use instead

Yeah I tried to find one in the `deltadental.com` source HTML but couldn't find one. Where did you find that one? I replaced the PNG image with that in 762d501ffef59e2bc11839eee87de220b17fa359.

**Comment by Carlgo11** [comment]
I just googled `deltadental.com filetype:svg` and found the image :shrug:

**Review by Carlgo11 — APPROVED** [review]
Thanks for your contribution! 🙂

**Pull request merged**